bin/downloadOmaRegistry.js

```diff
@@ -68,12 +67,38 @@ function parseRegistry(registryData, callback) {
             logger.info('Initiating data parse.');
             initiateData = true;
         } else if (initiateData) {
-            if ($($(this).find('td').get(1)).text().indexOf('urn') >= 0) {
+            if (
+                $(
+                    $(this)
+                        .find('td')
+                        .get(1)
+                )
+                    .text()
+                    .indexOf('urn') >= 0
+            ) {
                 var obj = {
-                    name: $($(this).find('td').get(2)).text(),
```

Comment:
I think the "compact" syntax is better in this point. Could we disable the rule that is causing this? Or add some kind of local exception using the proper `/* ... */` mark (if supported)?

Reply:
Added `prettier-ignore`. Fixed 46a4bcb